backend/docker-compose.yml

```diff
@@ -1,7 +1,52 @@
-version: '3'volumes:
-
+version: '3.4'
 
 services:
+  php:
```

Comment:
Nebylo by lepší tohle nazvat `app`?

Reply:
Vycházel jsem z originálního docker-compose z https://github.com/dunglas/symfony-docker/blob/main/docker-compose.yml. Ale klidně to můžeme upravit.